### DISCUSSION TITLE
ES module import?

### DISCUSSION THREAD

**Discussion by fpolli** [videojs/video.js#8926] [Q&A] [answered]
The documentation tells how to install video.js from npm, but I can find nowhere that tells me what import statement to use to add the player to my existing bundle. I have looked at the source of various examples and nowhere do I see that es modules are used. What would the import statement be?

**Comment by Thexasan** [comment, marked as answer]
To use Video.js in your project with ES modules, you can use the following import statement:
```js
import videojs from 'video.js';
```

**Reply by fpolli** [reply]
Thanks
